src/panels/lovelace/components/hui-image.ts

```diff
@@ -233,7 +311,7 @@ export class HuiImage extends LitElement {
     }
 
     // One the first render we will not know the width
```

Comment:
Should we wait on the first render then?

Reply:
If we did we can still get a 1.2MiB image on the first load since the 4k cameras produce very large images. The worse case now is ~120k and then about 50k on each subseqeuent.

Reply:
I mean wait loading the image until we know the width of the element? So we have no worse case at all?

We probably could also take the width of the element itself instead of the image element? That should be available at first render?